src/main/java/swervelib/SwerveDrive.java

```diff
@@ -56,9 +56,16 @@ public class SwerveDrive {
    */
   public final SwerveDriveConfiguration swerveDriveConfiguration;
   /**
-   * Swerve odometry.
+   * Swerve+vision pose estimator.
    */
   public final SwerveDrivePoseEstimator2 swerveDrivePoseEstimator;
+  /**
+   * Swerve-only pose estimator, without adding vision measurements to it. Used to calculate the
+   * effects of vision measurements, by comparing the poses estimated with and without vision. Don't
+   * use for anything else other than this. The name starts with underscrore to reduce the chance of
+   * listing this in the auto-completion suggestions.
+   */
+  private final SwerveDrivePoseEstimator2 _swerveDrivePoseEstimator_NoVision_;
```

Comment:
Just for a test, we can decline or remove this if it is not useful, but wanted to see how much vision helps correct our pose.
I am keeping another pose estimator which is not fed with the vision measurements (only with wheel odometry).
We calculate the distance between the two poses and report it on SmartDashboard.